docs/reference/filters.md

```diff
@@ -1209,6 +1224,20 @@ forwardToken("X-Tokeninfo-Forward")
 forwardToken("X-Tokeninfo-Forward", "access_token")
 ```
 
+## forwardTokenField
+
+The filter takes the comma-separated lists of header names and fields as its first and second arguments. The corresponding fields from the result of token info, token introspection or oidc user info are added as 
+corresponding headers when the request is passed to the backend. 
+
+If this filter is used when there is no token introspection, token info or oidc user info data
+then it does not have any effect.
+
+Examples:
+
+```
+forwardTokenField("X-Tokeninfo-Forward-Oid,X-Tokeninfo-Forward-Sub", "oid,sub")
```

Comment:
Looking at the current implementation this would produce 
* `X-Tokeninfo-Forward-Oid: <oidc value>`
* `X-Tokeninfo-Forward-Sub: <sub value>`

Lets simplify it and only support single header and field name - if users need multiple headers they could use sequence of filters: `forwardTokenField("X-Tokeninfo-Forward-Oid", "oid") -> forwardTokenField("X-Tokeninfo-Forward-Sub", "sub")`.